visualstudio/2017/netstandard/NetstandardModuleSample/NetstandardModuleSample/module_dev_sample.json

```diff
@@ -0,0 +1,55 @@
+﻿{
+    "loaders": [
+        {
+            "type": "dotnetcore",
+            "name": "dotnetcore",
+            "configuration": {
+                "binding.path": "dotnetcore.dll",
+                "binding.coreclrpath": "C:\\Program Files\\dotnet\\shared\\Microsoft.NETCore.App\\1.1.1\\coreclr.dll",
+                "binding.trustedplatformassemblieslocation": "C:\\Program Files\\dotnet\\shared\\Microsoft.NETCore.App\\1.1.1\\"
+            }
+        }
+    ],
+    "modules": [
+        {
+            "name": "logger",
+            "loader": {
+                "name": "native",
+                "entrypoint": {
+                    "module.path": "logger.dll"
+                }
+            },
+            "args": {
+                "filename": "log.txt"
```

Comment:
Please keep using same indention style for your commits: don't mix 2-spaces with 4-spaces. Use 2-spaces only.